docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md

```diff
@@ -11,19 +11,21 @@
    - To check your Windows version, run `winver` on the command line.
 
 ## Software prerequisites
+For specific versions we use for each release, see [install-windows-prereqs.ps1](https://github.com/openenclave/openenclave/blob/master/scripts/install-windows-prereqs.ps1#L6-L36).
 - [Microsoft Visual Studio Build Tools 2019](https://aka.ms/vs/15/release/vs_buildtools.exe)
 - [Git for Windows 64-bit](https://git-scm.com/download/win)
-- [Clang/LLVM for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/LLVM-8.0.1-win64.exe)
+- [Clang/LLVM 10.0.0 for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe)
 - [Python 3](https://www.python.org/downloads/windows/)
-- [ShellCheck](https://oejenkins.blob.core.windows.net/oejenkins/shellcheck-v0.7.0.zip)
-- [OpenSSL 1.1.1](https://oejenkins.blob.core.windows.net/oejenkins/openssl.1.1.1506.73.nupkg)
+- [ShellCheck](https://github.com/koalaman/shellcheck/releases)
```

Comment:
In my opinion, it is not helpful to just point to a general link without saying what version is needed, at least a minimum version.

Reply:
I've added a version to this reference. Thanks!